src/firebolt/client/auth.py

```diff
@@ -50,18 +68,40 @@ def __init__(
         self._expires: Optional[int] = None
 
     def copy(self) -> "Auth":
+        """Make another auth object with same credentials.
+
+        Returns:
+            Auth: Auth object
+        """
         return Auth(self.username, self.password, self._api_endpoint)
 
     @property
     def token(self) -> Optional[str]:
+        """Acquired bearer token.
+
+        Returns:
+            Optional[str]: Acquired token
+        """
         return self._token
 
     @property
     def expired(self) -> Optional[int]:
+        """Check if current token is expired.
+
+        Returns:
+            bool: True if expired, False otherwise
```

Comment:
Looks like it does return boolean, but our type hint is wrong.

Reply:
Hm, wondering how mypy didn't catch this